src/Sulu/Bundle/WebsiteBundle/Navigation/NavigationMapperInterface.php

```diff
@@ -36,7 +36,8 @@ public function getNavigation(
         $depth = 1,
         $flat = false,
         $context = null,
-        $loadExcerpt = false
+        $loadExcerpt = false,
+        $segmentKey = null
```

Comment:
Was this committed accidentally? Seems this change  has nothing todo with this PR?

Reply:
No, I've added it afterwards, because it was forgotten in another PR. That's how the interface is actually looking.